### PR TITLE
Update obvinf.hpp for v1/3

### PR DESCRIPTION
**Pull request by james-d-mitchell**

### CODE REVIEW

File: src/obvinf.cpp [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+#include "main.hpp"  // for init_obvinf
+
+namespace py = pybind11;
+
+namespace libsemigroups {
+
+  void init_obvinf(py::module& m) {
+    m.def(
+        "is_obviously_infinite",
+        [](Presentation<word_type> const& p) {
```
Comment (Joseph-Edwards) [inline src/obvinf.cpp:38]: Do we want to do this for `Presentation<std::string>` too?

Comment (james-d-mitchell) [reply src/obvinf.cpp:38]: Good point forgot that one!

File: src/knuth-bendix.cpp
Comment (Joseph-Edwards) [inline src/knuth-bendix.cpp:1]: Should this function be moved into `obvinf.cpp` to be with the rest of them?

File: src/knuth-bendix.cpp [hunk 2]
```diff
@@ -550,15 +550,32 @@ is returned.
...
+This function returns ``True`` if the quotient of the finitely presented
+semigroup or monoid defined by the :any:`KnuthBendix` object *kb* is obviously
+infinite; ``False`` is returned if it is not.
+
+:param kb: the :any:`KnuthBendix` instance.
+:type kb: KnuthBendix
+
+:returns:
+    Whether or not the quotient defined by a :any:`KnuthBendix` instance is
+    obviously infinite.
```
Comment (Joseph-Edwards) [inline src/knuth-bendix.cpp:573]:
```suggestion
:returns:
    Whether or not the quotient defined by a :any:`KnuthBendix` instance is
    obviously infinite.
:rtype:
    bool
```

**Review by Joseph-Edwards — CHANGES_REQUESTED** [review]
Happy to merge this modulo the few minor suggestions. Thanks @james-d-mitchell!

**Review by Joseph-Edwards — APPROVED** [review]

**Pull request merged**